docker/Dockerfile.pdo-build

```diff
@@ -122,20 +122,21 @@ RUN \
  && env | grep PDO | egrep -v 'PDO_ENCLAVE_CODE_SIGN_PEM|PDO_INSTALL_ROOT|PDO_HOME' | sed 's/^PDO/export PDO/g' >> /etc/profile.d/pdo.sh \
 # Build PDO usuing the standard build scripts
  && make -C /project/pdo/src/private-data-objects/build/ NO_SGX_RUN_DURING_BUILD=true \
- && echo '. /project/pdo/build/bin/activate' >> /etc/profile.d/pdo.sh \
- && . /project/pdo/build/bin/activate \
+# && echo '. /project/pdo/build/bin/activate' >> /etc/profile.d/pdo.sh \
+# && . /project/pdo/build/bin/activate \
 # - build script install in virtualenv. For docker this seems a bit overkill but easier than to
 #   duplicate all build instructions.
- && cd /project/pdo/src/private-data-objects \
- && cd python \
- && python3 setup.py build_ext \
- && python3 setup.py install \
+# Since PR #439 https://github.com/hyperledger-labs/private-data-objects/pull/439 these commands are called via the top-level build/Makefile
+# && cd /project/pdo/src/private-data-objects \
+# && cd python \
+# && python3 setup.py build_ext \
+# && python3 setup.py install \
 #
 # Build ledger interface.
- && cd /project/pdo/src/private-data-objects/eservice/ \
- && sed -i 's/python /python3 /g' Makefile \
- && make \
- && make install \
- && python3 setup.py install \
+# && cd /project/pdo/src/private-data-objects/eservice/ \
+# && sed -i 's/python /python3 /g' Makefile \
+# && make \
+# && make install \
+# && python3 setup.py install \
 
```

Comment:
please just delete these lines. if they are no longer necessary then remove them. Just make sure the commit comment explains *in a lot more detail than i see right now* why you can delete them.

Reply:
At this point, I'm going to wait for #438 to be merged before I come back to this.